### ISSUE TITLE
How to set the network proto for Thumos14?

### ISSUE THREAD

**Issue by jx-zhong-for-academic-purpose** [wanglimin/UntrimmedNet#11]
To adapt the network proto from ActivityNet1.2 to Thumos14, I have modified the fc_action and the consequent layers dimensions (100-->21). However, it goes wrong  when loading the trained model on Thumos14.  
Thus, I am wondering how to set the network proto for Thumos14, and whether it is convenient for you to release the network proto on Thumos14? 
Look forward to your feedbacks, and thank you for your kind attention to this matter.

**Comment by wanglimin** [comment]
We have added the deploy proto files of thumos.

**Comment by bityangke** [comment]
Could you share the proposal file(window shots) of thumos14?
Thanks very much！

**Comment by bityangke** [comment]
And also .solver file of thumos14 dataset.
Thanks very much for sharing your work.

**Comment by wanglimin** [comment]
We have uploaded the shot files of thumos. It should be noted these shot files are based on our own video decoding. If you decode videos differently, these shot files may not apply. Therefore, it is better to extract shot by yourself.

About solver file, it is almost the same with ANet solver except the iter numbers, which could be found in our paper.

The initial learning rate is set to 0.001 for the spatial stream and decreases every 4,000 iterations by a factor of 10, and the whole training stops at 10, 000 iterations. For the temporal stream, we set the initial learning rate to 0.005, which is decreased every 6,000 iterations by a factor of 10, and it stops training at 18, 000 iterations.
